generative_models/InfoGAN_pytorch/README.md

```diff
@@ -0,0 +1,155 @@
+# PyTorch implementation of InfoGAN [*Paper*](https://arxiv.org/pdf/1606.03657.pdf)
+## Requirements
+* numpy
+* python
+* pytorch
+* pytorch-vision
+* pickle
+* cuda(*highly recommended*)
+## Usage 
+* Run the following command : 
+      <pre><code>python train.py --num_epoch <num_of_epochs> --batch_size <batch_size> --num_workers <num_workers> --lrD <learning_rate_discriminator> --lrG <learning_rate_generator> --beta1 <beta1_adam_param> --beta2 <beta2_adam_param> --recog_weight <recogniser_weight> --save_epoch <model_checkpoints>
+      </code></pre>
+> ***NOTE** : Use python3 instead in case of Linux/Mac .*
+* Arguments
+  * -num_epoch : Default: 50, number of epochs.
+  * -batch_size : Default: 100.
+  * -num_workers : Default : 2.
+  * -lrD : Adam optimizer discriminator learning rate, default : 2e-4 (0.0002)
+  * -lrG : Adam optimizer generator learning rate, default : 1e-3 (0.001)
+  * -beta1 : Default : 0.5
+  * -beta2 : Default : 0.99
+  * -recog_weight : Default : 0.5, coefficient of information maximising.
+  * -model_path : Default : 'trained_model'+ current datetime (*added itself*)
+  * -save_epoch : Default : 5.
```

Comment:
you can simply use `python main,py -h`, this will return argument list in much more fashionable way.

Reply:
and write default values separately??

Reply:
the default value will also be shown, in `python main.py -h`. Also, there is argument called help in parser.add_argument which you can use to describe the usage of argument.